php/Terminus/Models/Environment.php

```diff
@@ -361,6 +357,11 @@ public function createBackup($args) {
       'files'      => isset($args['files']),
       'ttl'        => $ttl
     );
+
+    $options  = array('environment' => $this->get('id'), 'params' => $params);
+    $workflow = $this->site->workflows->create('do_export', $options);
+    $workflow->wait();
```

Comment:
Not a blocker, but we should try to get this `wait()` call out of the model and into the command file eventually.